### ISSUE TITLE
In Lenovo Daydream devices,controller is getting disconnected sometime.

### ISSUE THREAD

**Issue by talk2saroj** [Samsung/GearVRf#2030]
In Lenovo Daydream devices, sometimes controller is not working.
Is there any way to get some call back whenever the controller is detached, so that we can connect it back.
Currently, I am initializing the cursor controller only once inside OnInit() method of GVRMain class using the following code.

public void initCursorController() {
mainScene.getEventReceiver().addListener(mTouchHandler);
GVRInputManager inputManager = context.getInputManager();
inputManager.selectController(new GVRInputManager.ICursorControllerSelectListener() {
public void onCursorControllerSelected(GVRCursorController newController, GVRCursorController oldController) {
if (oldController != null) {
oldController.removePickEventListener(mTouchHandler);
}
mCursorController = newController;
newController.addPickEventListener(mTouchHandler);
newController.setCursor(createCursor());
newController.setCursorDepth(-CURSOR_DEPTH);
newController.setCursorControl(GVRCursorController.CursorControl.PROJECT_CURSOR_ON_SURFACE);
}
});
}

**Comment by liaxim** [comment]
Closing as a duplicate of https://github.com/Samsung/GearVRf/issues/2029
